src/constants/route-onboard/type.ts

```diff
@@ -0,0 +1,15 @@
+import { Graphic } from '@/components/graphic/type';
+
+export type TitleType = {
+  [index: string]: { title: React.ReactElement };
+};
+
+export type ItemType = {
+  id: number;
+  name: string;
+  emoji: Graphic;
+};
+
+export type ItemsType = {
+  [index: string]: ItemType[];
+};
```

Comment:
```suggestion
export type ItemsType = {
  DAIRY: ItemType[];
  TRAVEL: ItemType[];
  EXERCISE: ItemType[];
};
```

지금은 key로 아무 문자열이나 들어 갈 수 있는데, 이렇게 선언하는 것은 어떨까요?